python_cli/liquidai_cli/commands/stack.py

```diff
@@ -0,0 +1,136 @@
+"""Stack management commands."""
+
+import typer
+from pathlib import Path
+from liquidai_cli.utils.docker import DockerHelper
+from liquidai_cli.utils.config import load_config, extract_model_name
+from liquidai_cli.utils.prompt import confirm_action
+
+app = typer.Typer(help="Manage the on-prem stack")
+docker_helper = DockerHelper()
+
+
+@app.command()
+def launch(
+    upgrade_stack: bool = typer.Option(False, "--upgrade-stack", help="Upgrade stack version"),
+    upgrade_model: bool = typer.Option(False, "--upgrade-model", help="Upgrade model version"),
+):
+    """Launch the on-prem stack."""
+    config = load_config()
+
+    if upgrade_stack:
+        config["stack"]["version"] = "c3d7dbacd1"
+    if upgrade_model:
+        config["stack"]["model_image"] = "liquidai/lfm-7b-e:0.0.1"
+
+    # Set model name
+    model_image = config["stack"]["model_image"]
+    model_name = f"lfm-{extract_model_name(model_image)}"
+    config["stack"]["model_name"] = model_name
+
+    # Generate environment file for docker-compose
+    env_vars = {
+        "JWT_SECRET": config["stack"]["jwt_secret"],
+        "API_SECRET": config["stack"]["api_secret"],
+        "AUTH_SECRET": config["stack"]["auth_secret"],
+        "STACK_VERSION": config["stack"]["version"],
+        "MODEL_IMAGE": config["stack"]["model_image"],
+        "MODEL_NAME": config["stack"]["model_name"],
+        "POSTGRES_DB": config["database"]["name"],
+        "POSTGRES_USER": config["database"]["user"],
+        "POSTGRES_PASSWORD": config["database"]["password"],
+        "POSTGRES_PORT": str(config["database"]["port"]),
+        "POSTGRES_SCHEMA": config["database"]["schema"],
+        "DATABASE_URL": (
+            f"postgresql://{config['database']['user']}:{config['database']['password']}"
+            f"@liquid-labs-postgres:{config['database']['port']}/{config['database']['name']}"
+        ),
+    }
+
+    # Write environment variables to .env for docker-compose
+    with open(".env", "w") as f:
+        for key, value in env_vars.items():
+            f.write(f"{key}={value}\n")
```

Comment:
Is it necessary to create `.env` for `docker-compose`? Can it consume environment variable from the environment?